crates/apub/src/lib.rs

```diff
@@ -49,13 +49,13 @@ impl UrlVerifier for VerifyUrlData {
         ..
       } => anyhow!("Federation disabled"),
       LemmyError {
-        error_type: LemmyErrorType::DomainBlocked(_),
+        error_type: LemmyErrorType::DomainBlocked(domain),
         ..
-      } => anyhow!("Domain is blocked"),
+      } => anyhow!("Domain {domain:?} is blocked"),
```

Comment:
I think the `:?` is for debug formatting, any reason why thats needed? Otherwise looks good.

Reply:
It's intentional. Mostly it's just to put quotes around the domain. But it also ensures that any "weird" characters in the domain would be shown explicitly and not printed as they are. But not sure if such characters ever appear in the domain.